commands/new_function.go

```diff
@@ -60,35 +59,30 @@ the "Dockerfile" lang type in your YAML file.
 	}
 
 	if len(args) < 1 {
-		fmt.Println("Please provide a name for the function")
-		return errors.New("please provide a name for the function")
+		return fmt.Errorf("please provide a name for the function")
 	}
 	functionName = args[0]
 
 	if len(lang) == 0 {
-		fmt.Println("You must supply a function language with the --lang flag")
-		return errors.New("you must supply a function language with the --lang flag")
+		return fmt.Errorf("you must supply a function language with the --lang flag")
 	}
 
 	PullTemplates("")
 
 	if validTemplate(lang) == false {
-		fmt.Printf("%s is unavailable or not supported.\n", lang)
-		return errors.New("language is unavailable or not supported")
+		return fmt.Errorf("%s is unavailable or not supported", lang)
 	}
 
 	if _, err := os.Stat(functionName); err == nil {
-		fmt.Printf("Folder: %s already exists\n", functionName)
-		return errors.New("folder already exists")
+		return fmt.Errorf("Folder: %s already exists", functionName)
 	}
 
 	if err := os.Mkdir("./"+functionName, 0700); err == nil {
 		fmt.Printf("Folder: %s created.\n", functionName)
 	}
 
 	if err := updateGitignore(); err != nil {
-		fmt.Println("Got unexpected error while updating .gitignore file.")
-		return err
+		return fmt.Errorf("Got unexpected error while updating .gitignore file: %s", err)
```

Comment:
should begin with lowercase character

(see https://github.com/golang/go/wiki/CodeReviewComments#error-strings)